README.md

```diff
@@ -16,3 +16,8 @@ To see how it looks like, you can access the latest stable version at : http://q
 ### Android
 
 The Android app uses a webview on the "web app", adding easiness of use, especially when network data is not available.
+
+
+### Delta/paragling specific questions for winch
+
+The fork allows winch questions (tagged with T letter)
```

Comment:
This change in README is not necessary IMHO.
Your addition would not be a fork, but integrated mainsteam.